main.tf

```diff
@@ -147,6 +147,7 @@ module "build" {
   aws_account_id     = "${signum(length(var.aws_account_id)) == 1 ? var.aws_account_id : data.aws_caller_identity.default.account_id}"
   image_repo_name    = "${var.image_repo_name}"
   image_tag          = "${var.image_tag}"
+  github_token       = "${var.github_oauth_token}"
```

Comment:
You should use `github_token`. https://github.com/cloudposse/terraform-aws-codebuild/pull/11

Reply:
No. Because https://github.com/cloudposse/terraform-aws-cicd/blob/master/variables.tf#L27 
This is the same token